Homework7/Polaroid.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Homework7
+{
+    public class Polaroid : ItemWorkingWithPaper, IPhoto
+    {
+        public double numberOfPixelsInCamera;
```

Comment:
Make fields private, that's why we use public properties. Or even you can use only properties.
Naming - private fields starts with "_"+lower case letter - e.g. private string _name (not _Name)

Reply:
fixed (hope so)